app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -68,12 +68,18 @@ class WallpaperSettingsFragment : Fragment() {
                                 onWallpaperSelected(it, result, this@apply)
                             }
                         },
-                        onLearnMoreClick = { url ->
+                        onLearnMoreClick = { url, collectionName ->
                             (activity as HomeActivity).openToBrowserAndLoad(
                                 searchTermOrURL = url,
                                 newTab = true,
                                 from = BrowserDirection.FromWallpaper,
                             )
+                            Wallpapers.learnMoreLinkClick.record(
+                                Wallpapers.LearnMoreLinkClickExtra(
+                                    url = url,
+                                    collectionName = collectionName,
```

Comment:
thinking out loud, I'm wondering what the significance is of capturing the collection name _and_ the url

Reply:
This wasn't a requirement, but I thought it might make it easier to query the data. I assumed it would be more likely someone would remember the name of a collection off the top of their head than an URL. If that doesn't seem like a typical usage than I'm happy to remove it